input/fsh/instances.fsh

```diff
@@ -0,0 +1,668 @@
+InstanceOf: DkCorePatient
+Title: "Example of valid patient with multiple names"
+Description: "Example of valid patient with multiple names"
+Usage: #example
+* name[0].use = #official
+* name[0].family = "Meyer"
+* name[0].given[0] = "Niels"
+* name[0].given[1] = "Christian"
+* name[0].prefix[0] = "Mr"
+* name[1].use = #usual
+* name[1].given[0] = "Bubber"
+* identifier[0].system = "urn:oid:1.2.208.176.1.2"
+* identifier[0].value = "1112640001"
+* gender = #male
+* birthDate = "1964-12-11"
+* generalPractitioner[0].identifier.system = "urn:oid:1.2.208.176.1.1"
+* generalPractitioner[0].identifier.value = "487341000016005"
+* generalPractitioner[0].display = "Charlottenlund Lægehus"
+
+Instance: mogensen
+InstanceOf: DkCorePatient
+Title: "Example of valid patient with danish marital status"
+Description: "Example of valid patient with danish marital status"
+Usage: #example
+* name[0].use = #official
+* name[0].family = "Mogensen"
+* name[0].given[0] = "Jan"
+* name[0].prefix[0] = "Mr"
+* maritalStatus = $dk-marital-status#P "Registreret partnerskab"
+* identifier[0].system = "urn:oid:1.2.208.176.1.2"
+* identifier[0].value = "1112640001"
+* gender = #male
+* birthDate = "1964-12-11"
+
+Instance: 283
+InstanceOf: DkCorePatient
+Title: "Example of valid patient with full address"
+Description: "Example of valid patient with full address"
+Usage: #example
+* identifier[0].use = #official
+* identifier[0].system = "urn:oid:1.2.208.176.1.2"
+* identifier[0].value = "0506504005"
+* name[0].use = #official
+* name[0].family = "Lind"
+* name[0].given[0] = "Torsteinn"
+* gender = #male
+* birthDate = "1950-06-04"
+* address[0].extension[0].url = "http://hl7.dk/fhir/core/StructureDefinition/dk-core-municipalityCodes"
+* address[0].extension[0].valueCodeableConcept = $dk-core-municipality-codes#0330
+* address[0].extension[1].url = "http://hl7.dk/fhir/core/StructureDefinition/dk-core-RegionalSubDivisionCodes"
+* address[0].extension[1].valueCodeableConcept = $dk-core-regional-subdivision-codes#DK-85
+* address[0].use = #home
+* address[0].type = #postal
+* address[0].line[0] = "Nordre Ringgade 3"
+* address[0].city = "Frederiksberg C"
+* address[0].postalCode = "8000"
+* address[0].country = "DK"
+* maritalStatus = $v3-MaritalStatus#U "unmarried"
+* generalPractitioner[0].identifier.system = "urn:oid:1.2.208.176.1.4"
+* generalPractitioner[0].identifier.value = "79641000016006"
+* generalPractitioner[0].display = "Peter Sønderby"
+
+Instance: ukendt
+InstanceOf: DkCorePatient
+Title: "Example of valid patient unknown identifier registry"
+Description: "Example of valid patient unknown identifier registry. The patient is a 'John Doe' with an Identifier from an unofficial register"
+Usage: #example
+* name[0].use = #temp
+* name[0].family = "Ukendt"
+* name[0].given[0] = "Anders"
+* identifier[0].system = "http://rn.dk"
+* identifier[0].value = "0706830VJ9"
+* gender = #male
+* birthDate = "1983-06-07"
+
+Instance: administrative-gender-supplement
```

Comment:
Why is this and other code systems / value sets defined as instances and not the native CodeSystem / ValuSet in FSH?

Reply:
because its from the supplement